lib/rules/alpha-value-notation/index.mjs

```diff
@@ -112,7 +112,10 @@ const rule = (primary, secondaryOptions) => {
 					endIndex,
 					result,
 					ruleName,
-					fix,
+					fix: {
+						apply: fix(),
```

Comment:
if you revert the change l102 or rename the function l102, I suggest either

```suggestion
						apply: fix,
```
or
```suggestion
						apply,
```